src/NTS.Application/DataTransferObjects/Models/CommonModel.cs

```diff
@@ -0,0 +1,434 @@
+﻿using Not.Exceptions;
+using Not.Random;
+using NTS.Domain.Aggregates;
+using NTS.Domain.Core.Aggregates;
+using NTS.Domain.Core.Aggregates.Participations;
+using NTS.Domain.Enums;
+using NTS.Domain.Objects;
+using NTS.Domain.Settings;
+using static NTS.Application.DataTransferObjects.Models.CoreModel;
+using static NTS.Application.DataTransferObjects.Models.SetupModel;
+
+namespace NTS.Application.DataTransferObjects.Models;
+
+public class CommonModel
```

Comment:
Refactor models as dicussed - `CommonModel`, `SetupModel` and `CoreModel` are superfluous. Segregate the models by usage to ensure you don't have unnecessary properties. See following example from my current PR:
- [CoreAthleteModel](https://github.com/Not-Endurance/not-timing-system/pull/481/changes#diff-cea91643e2e55c5f185d63fd23dd0cb91428b6f7067da9b78a53f42365227695L9)
- [SetupAthleteModel](https://github.com/Not-Endurance/not-timing-system/pull/481/changes#diff-c0bd59e142e5e001e43ecce81e3c775b9770022197a1705e47bb0cb002b7df31R7)

This way you can also avoid having to inherit the `Document` abstract class. I've left the name `SetupAthleteDocument`, but you can rename that to `SetupAthleteModel`